### PR TITLE
chore: asf.yaml to conform to apache specifications

### PR DESCRIPTION
**Pull request by KeHaohaoke**
Signed-off-by: KeHaoKH <hao.ke@merico.dev>

# Summary

asf.yaml to conform to apache specifications

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  features:
+    issues: false
+    projects: false
+    wiki: false
+  labels:
+    - devlake
+  enabled_merge_buttons:
+    squash:  true
+    merge:   false
+    rebase:  false
```
Comment (klesh) [inline .asf.yaml:30]: we need this to be `true`

File: .asf.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+publish:
```
Comment (klesh) [inline .asf.yaml:32]: I dont think we need this section. Looks like it is for website publication, which we dont need for this repo

File: .asf.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+github:
+  description: "Apache Devlake Website"
```
Comment (klesh) [inline .asf.yaml:19]: @Startrekzky please advise a better description

**Review by klesh — CHANGES_REQUESTED** [review]

File: .asf.yaml (hunk 1 above)
Comment (KeHaohaoke) [reply .asf.yaml]: > we need this to be `true`

rebase : true

File: .asf.yaml (hunk 3 above)
Comment (KeHaohaoke) [reply .asf.yaml]: > @Startrekzky please advise a better description

new content submit

File: .asf.yaml (hunk 2 above)
Comment (KeHaohaoke) [reply .asf.yaml]: > I dont think we need this section. Looks like it is for website publication, which we dont need for this repo

delet publish section

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**